Scripts/script-ADGetUser.yml

```diff
@@ -32,6 +34,11 @@ script: |-
   if demisto.get(demisto.args(), 'limit'):
       limit = demisto.args()['limit']
 
+
+  def escapeSpecialCharacters(s):
```

Comment:
what happens if there are two agencent brackets?
![image](https://user-images.githubusercontent.com/10514677/38408908-0a97d230-3988-11e8-8970-3183ad3c4ac5.png)
  
it seems that only the second `(` is missing escape characters..isn't it?